src/ethereum/frontier/utils/address.py

```diff
@@ -32,3 +34,26 @@ def to_address(data: Union[Uint, U256]) -> Address:
         The obtained address.
     """
     return data.to_be_bytes32()[-20:]
+
+
+def compute_contract_address(address: Address, nonce: Uint) -> Address:
+    """
+    Computes address of the new account that needs to be created.
+
+    Parameters
+    ----------
+    address :
+        The address of the account that wants to create the new account.
+    nonce :
+        The transaction count of the account that wants to create the new
+        account.
+
+    Returns
+    -------
+    address: `ethereum.frontier.eth_types.Address`
+        The computed address of the new account.
+    """
+    computed_address = keccak256(rlp.encode([address, nonce]))
+    canonical_address = computed_address[-20:]
+    padded_address = canonical_address.rjust(20, b"\x00")
```

Comment:
What does this line do? `computed_address` is always 32 bytes (because it is a 32 byte keccak), `canonical_address` always chops it down to 20 bytes, so padding to 20 bytes is always a no-op.